test/sandbox.test.ts

```diff
@@ -138,7 +138,23 @@ describeIf(
     });
 
     describe("sessions.authenticate", () => {
-      test("success", () => {
+      test("success", async () => {
+        // Make sure there's a key that can be used to sign the sandbox JWT.
+        //
+        // You should never need to trigger this for a real project in either environment (Test or
+        // Live). Signing keys will be automatically generated for your project the first time the
+        // API needs one to sign a JWT. Those keys will be automatically rotated as you continue
+        // to use sessions.
+        //
+        // We only need this in this test suite because it's a very narrow edge case:
```

Comment:
It might make a lot more sense for future readers to just write this as a test of `client.session.jwks` producing the intended result, instead of talking about the internals of Sandboxing

Reply:
That's a good point. I wanted it to block the call to `client.sessions.authenticate`, and I didn't know whether Jest would allow running tests in parallel. I was worried that someone would think they need this if they were using these tests as example code.

I was having trouble spelling the vague assert I need to do in callback form, but `async` made it readable again: a0527adddc4864646524874eaf4341a53731cbb7

Thanks!